distribution/changelog.txt

```diff
@@ -41,6 +41,7 @@
 - Change: [#16493] Boat Hire and Submarine Ride support costs now match their visual appearance.
 - Change: [#16710] Changed default view of Guest List to 'Thoughts' and selected tab will default to 'Summarised' (when opened from the menu).
 - Change: [#16859] Guests with umbrellas no longer always avoid going into a Maze.
+- Change: [#16912] Guests no longer perform slow motion jumps in Maze.
```

Comment:
This reads as if they jump at regular speed now, but the change prevents slow peeps from jumping at all.

Reply:
I can understand the confusion! I changed it, hopefully this is better.